netbox/utilities/forms/bulk_import.py

```diff
@@ -90,8 +90,17 @@ def _clean_csv(self, data):
         """
         Clean CSV-formatted data. The first row will be treated as column headers.
         """
+
+        # Determine the CSV dialect
```

Comment:
We should add some indication to the user that semicolons can also be used as delimiters now. Maybe on the form field's help text? It's also worth calling out in the documentation.